src/module-2/card/index.js

```diff
@@ -67,7 +57,23 @@ export default class Card {
         <span>ADD TO CART</span>
     </button>
   </div>`
-    return product
+  }
+  render() {
+    const product = document.createElement('article');
+    product.classList.add('product-card')
+    const rating = Number(this.rating)
```

Comment:
I hope you understand that if `this.rating` is `null` then it will be converted to 0.